src/components/Card/Card.jsx

```diff
@@ -5,10 +5,18 @@ import CardImage from '../CardImage/CardImage';
 import CardFooter from '../CardFooter/CardFooter';
 
 const Card = ({ img, title, subtitle, label }) => {
+  const deleteHandler = () => {};
+  const editHandler = () => {};
```

Comment:
I think that `Card` component can be presentational thus event handlers should be passed from parent component as props. 

The first benefit from it is that with big number of cards on the page we will have only two handlers in parent component rather than two handlers for every card.
The second one is that presentational component must not know about how actions are being handled, the only purpose of `Card` component is to show some UI.

Does it make sense? 🤔

Reply:
Agree. I just forgot to remove it. Handlers will be assigned directly to Buttons in CardFooter from MianContext